src/utils/get-config.js

```
@@ -0,0 +1,16 @@
+import { readFileSync } from 'fs';
+import path from 'path';
+import resolveFileUrl from './resolve-file-url';
+
+
+export default function (appPath) {
+    var configPath   = path.join(appPath, '.testcafe-electron-rc');
+    var configString = readFileSync(configPath).toString();
```

I think we should throw an error with message like `Place .testcafe-electron-rc file to the root of your application`  if the config files doesn't exist